js/apps.js

```diff
@@ -0,0 +1,78 @@
+function App(name, logo, href) {
+    this.title = name;
+    this.logo = logo;
+    this.href = href;
+}
+
+var DOMAIN = "https://channeli.in/";
+
+buysell = new App("Buy & Sell", "buysell", "buyandsell");
```

Comment:
you are using global declarations. Will throw error when google implements 'use-strict' by default. Better change this to `var buysell`